build-go.sh

```diff
@@ -50,14 +50,18 @@ function doBuild() {
 	local goarch=$2
 	local target=$3
 	local output=$4
+  local version=$5
+
+  dir=$output
+	name=$(echo $target | awk -F'/' '{ print $3 }')
+  binname=${name}_${version}_${goos}-${goarch}
 
 	echo "==> building for $goos $goarch"
 
-	dir=$output/$1-$2
-	if [ -e $dir ]; then
-		echo "    $dir exists, skipping build"
-		return
-	fi
+	# if [ -e $dir ]; then
+	# 	echo "    $dir exists, skipping build"
```

Comment:
mmm, if we're going to force a rebuild every time, maybe just have a flag that deletes the dir. I don't like the idea of building in dirty dirs